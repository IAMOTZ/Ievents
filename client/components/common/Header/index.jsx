import React from 'react';
import PropTypes from 'prop-types';

const headerStyle = {
  backgroundColor: 'rgba(0,0,0, 0.9)'
};

const Header = props => (
  <nav className="navbar w-100 mt-3 d-none d-lg-block" style={headerStyle} id="header">
    <strong className="navbar-brand text-white">{props.text}</strong>
  </nav>
);

Header.defaultProps = {
  text: null,
};
Header.propTypes = {
  text: PropTypes.string,
};

export default Header;
