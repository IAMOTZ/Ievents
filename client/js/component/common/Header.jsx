import React from 'react';

const Header = props => (
  <nav className="navbar w-100 mt-3 d-none d-lg-block">
    <a className="navbar-brand text-white">
      <strong>{props.text}</strong>
    </a>
  </nav>
);

export default Header;
