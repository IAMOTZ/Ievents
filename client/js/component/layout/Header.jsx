import React from 'react';

export default (props) => {
  return (
    <nav className="navbar w-100 mt-3 d-none d-lg-block">
      <a className="navbar-brand text-white">
        <strong>{props.text}</strong>
      </a>
    </nav>
  );
};
