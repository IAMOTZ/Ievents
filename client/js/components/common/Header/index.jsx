import React from 'react';
import PropTypes from 'prop-types';

const Header = props => (
  <nav className="navbar w-100 mt-3 d-none d-lg-block">
    <strong className="navbar-brand text-white">{props.text}</strong>
  </nav>
);

Header.defaultProps = {
  text: null,
};
Header.propTypes = {
  text: PropTypes.string,
};

export default Header;
