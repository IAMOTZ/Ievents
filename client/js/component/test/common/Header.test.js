import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../../common/Header.jsx';


describe('<Header />', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Header text="Testing Header"/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
