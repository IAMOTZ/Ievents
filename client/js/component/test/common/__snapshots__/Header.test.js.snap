// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<nav
  className="navbar w-100 mt-3 d-none d-lg-block"
>
  <a
    className="navbar-brand text-white"
  >
    <strong>
      Testing Header
    </strong>
  </a>
</nav>
`;
